.github/workflows/android-build.yml

```diff
@@ -15,27 +15,28 @@ jobs:
     name: Check spotless
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: "temurin"
           java-version: 11
-          cache: 'gradle'
       - name: Check formatting using spotless
-        run: ./gradlew spotlessCheck --stacktrace
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: spotlessCheck --stacktrace
```

Comment:
https://github.com/gradle/gradle-build-action

Ref https://github.com/detekt/detekt/pull/3723, it's really so fast to run jobs, and many other advantages.